src/EventTimer.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { EventTimer } from './EventTimer';
+import { trackEvent } from './GoogleAnalytics';
+
+jest.mock('./GoogleAnalytics');
+
+describe('EventTimer', () => {
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access -- delete performance to mock it as is readonly
+	delete (window as any).performance;
```

Comment:
GitHub Actions is now flagging this on every new PR as a linting error. Is there any way we could fix it or ask to ignore it?

Reply:
removed here: https://github.com/guardian/commercial-core/pull/176/files